### DISCUSSION TITLE
deploying a website

### DISCUSSION THREAD

**Discussion by satyammishra2407** [community/community#51748] [New to GitHub]
The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.   

my github repostiory : https://satyammishra2407.github.io/finalpro/

please help me

**Comment by queenofcorgis** [comment]
Hi there - could you elaborate more on what you are experiencing - when I click the link (https://satyammishra2407.github.io/frtProject.github.io/) in your repository to your Page - I'm able to access it!
